.travis.yml

```diff
@@ -3,7 +3,6 @@ env:
   - MAVEN_VERSION=3.0.5
   - MAVEN_VERSION=3.3.9
 jdk:
-  - oraclejdk7
```

Comment:
Ref. https://github.com/travis-ci/travis-ci/issues/7884

Reply:
Best to file that as a separate PR for speedy merging.